.github/workflows/e2e-with-release.yaml

```diff
@@ -0,0 +1,128 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Test proxy requests through the admission sidecar using release
+
+on:
+  pull_request:
+    branches: [ 'main' ]
+
+defaults:
+  run:
+    shell: bash
+
+permissions: read-all
+
+jobs:
+  e2e:
+    name: Proxy e2e test using release
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        k8s-version:
+        - v1.22.x
+        - v1.23.x
+        - v1.24.x
+
+    env:
+      KNATIVE_VERSION: "1.5.0"
+      POLICY_CONTROLLER_VERSION: "v0.2.0"
```

Comment:
should we use the 0.2.1 release (https://github.com/sigstore/policy-controller/releases/tag/v0.2.1) ?

Reply:
Yep, great call!